src/components/map/map.module.css

```diff
@@ -51,6 +51,13 @@
   width: 100%;
 }
 
+.borderRadius__top canvas {
```

Comment:
Probably doesn't matter in the slightest, but would it be better to have the radius on the container and `overflow: hidden;`? Probably not, same same.

Reply:
I think that's better 👍 Let me know if you can think of a better CSS class name...